python/plugins/processing/tests/testdata/qgis_algorithm_tests3.yaml

```diff
@@ -915,6 +915,22 @@ tests:
         name: expected/field_calculator_points.gml
         type: vector
 
+  - algorithm: native:fieldcalculator
+    name: Test field calculator points (new API)
+    params:
+      FIELD_LENGTH: 10
+      FIELD_PRECISION: 3
+      FIELD_TYPE: 0
+      FORMULA: "\"id2\" *2"
+      INPUT:
+        name: points.gml
+        type: vector
+      NEW_FIELD_NAME: test
```

Comment:
@suricactus the old algorithm used `FIELD_NAME` as input, is there a safe way to have this backwards compatible?

Reply:
Did you encounter an issue? This should keep the backwards compatibility. The old test is passing too? 
https://github.com/qgis/QGIS/pull/38695/files#diff-53600f4cb9f762166bd6d0f4d56f555cR125-R135